README.md

```diff
@@ -29,7 +29,8 @@
 - YAML
 - JSON
 - TOML
-- HTTP/S URL (Supports content based on the Content-Type header matching the MIME types of supported file formats in this list)
+
+***Note**: Supported file formats can be loaded from both local files and web URLs. For web URLs, ensure the response's Content-Type matches the file format's MIME type. Environment variable file types are not supported yet.*
```

Comment:
I'd remove this section "Environment variable file types are not supported yet"
It's a bit confusing, and it's a use-case that probably won't happen. I'm not even sure there's a MIME type for env files.